vignettes/scanpy_integration.ipynb

```diff
@@ -101,12 +112,26 @@
     "adata.var_names = gene_df[\"gene\"].values\n",
     "adata.obs_names = metadata.index.values\n",
     "\n",
-    "adata.obs = metadata\n",
-    "adata.X_pca = pca_df.values\n",
-    "adata.obsm[\"X_pca\"] = adata.X_pca\n",
+    "adata.obs = metadata.copy()\n",
+    "\n",
+    "# Depends on which PCA you loaded\n",
+    "adata.X_pca = pseudopca_df.values\n",
+    "adata.obsm['X_pca'] = pseudopca_df.values\n",
+    "\n",
+    "# Name according to embedding you saved\n",
+    "adata.X_umap = embedding_df.values\n",
+    "adata.obsm['X_umap'] = embedding_df.values\n",
     "\n",
     "adata.uns['neighbors'] = dict(connectivities=graph_conn_mtx.tocsr(), distances=graph_dist_mtx.tocsr())\n",
-    "adata.uns['neighbors']['params'] = dict(n_pcs=pca_df.shape[1], use_rep='X_pca', metric='cosine', method='umap', n_neighbors=30)"
```

Comment:
Why was this removed again? That would reintroduce the error from #30

Reply:
@GMaciag

Reply:
That's a mistake on my part. It should not have been deleted.

Reply:
Restored in 12f813f

Reply:
Thanks @GMaciag!